konflux/rpms.in.yaml

```diff
@@ -0,0 +1,4 @@
+packages: [jq] 
```

Comment:
Can you explain how these files are used for this PR?

I don't see anything using jq in this patch.

Reply:
The bundle updates I removed will eventually need these.